#def stuff():
    #pass

def int main():
    int x = 5
    return x

# This was erroring becase the second line was being treated as an indent.
# RET: 0
